src/sparseml/pytorch/sparsification/quantization/modifier_quantization.py

```diff
@@ -494,38 +551,105 @@ def _enable_module_qat(self, module: Module):
                     )
                 )
             module_fuse_fn(**self._model_fuse_fn_kwargs)
-        elif self._model_fuse_fn_name is None:  # default auto fn
-            self._model_fuse_fn_kwargs["inplace"] = True
-            fuse_module_conv_bn_relus(module, **self._model_fuse_fn_kwargs)
 
-        activation_qconfig_kwargs = self._get_updated_activation_qconfig_kwargs()
+        # build list of layer types that should not quantize output activations
+        to_remove_layer_name = []
+        if not self._quantize_linear_activations:
+            to_remove_layer_name.extend(["Linear", "LinearReLU"])
+
+        if not self._quantize_conv_activations:
+            to_remove_layer_name.extend(
+                [
+                    "Conv1d",
+                    "Conv2d",
+                    "Conv3d",
+                    "ConvBn1d",
+                    "ConvBn2d",
+                    "ConvBn3d",
+                    "ConvReLU1d",
+                    "ConvReLU2d",
+                    "ConvReLU3d",
+                    "ConvBnReLU1d",
+                    "ConvBnReLU2d",
+                    "ConvBnReLU3d",
+                ]
+            )
+        if len(to_remove_layer_name) == 0:
+            to_remove_layer_name = None
+
+        # fix for freezing batchnorm statistics when not fusing BN with convs.
+        # pytorch only supports freezing batchnorm statistics for fused modules.
+        # this fix wraps BN modules adding with a new module class that supports
+        # methods related to freezing/unfreezing BN statistics.
+        configure_module_bn_wrappers(module)
+
+        # set qconfig.
+        # if tensorrt flag is used, set activation and weights to symmetric
+        # quantization.
+        # otherwise, use the default values set in get_qat_qconfig
+        if self.tensorrt:
+            _symmetric_activations = True
```

Comment:
nit: prefer to only use leading `_` in names for class/module names that are meant to be private

Reply:
Resolved.